### ISSUE TITLE
not work

### ISSUE THREAD

**Issue by cepera2412** [apaex/PAJ7620-ESPHome#3]
first added to my config.
then completely rewrote the code as here. only with your network settings, etc.
at the end of the firmware gives an error:
[E]  [paj7620:034]     Read ERROR, code: 2

**Comment by apaex** [comment]
Your sensor is not responding. This is either an incorrect connection, an incorrect address, or a faulty sensor. To begin with, check the work without ESP Home by downloading a clean example from the seeed-studio/Gesture PAJ7620 library and get the values from the sensor.

**Comment by cepera2412** [comment]
maybe the problem is because i have paj7620u2?  all pinouts are the same

**Comment by apaex** [comment]
No, there shouldn't be a difference. Check on a clean library https://github.com/Seeed-Studio/Gesture_PAJ7620 , its examples should work. If they do not work, then the sensor is faulty or connected incorrectly

**Comment by cepera2412** [comment]
downloaded the library for arduino and the sketch is simple, checked, it works.  maybe the problem needs to be 5v, not 3v3 ?!  I connected it to arduino according to the instructions for 5v.  or the second option: I threw the library into esphome in the wrong place.  I put it in config/esphome/paj7620_.h.  There I have the project file itself .yaml

**Comment by apaex** [comment]
My sensor works well from 3.3 V. But maybe your version of the board requires 5 V. Send a close-up photo of your sensor, I'll try to understand. But it seems that this is the case

**Comment by cepera2412** [comment]
![Uploading IMG_20230116_132015.jpg…]()

**Comment by cepera2412** [comment]
![IMG_20230116_132015](https://user-images.githubusercontent.com/86368554/212666978-7ce8807f-9dcd-429c-b900-9ed9354f7213.jpg)

**Comment by cepera2412** [comment]
I set the power supply to 3v3 and 5v.  connected as shown in the picture.  paj7620_.h threw config into esphome.  even copied your full code and the same error.  stitched another sensor, movement.  all right.  working.
